### PR TITLE
Added header css including the navbar style and .btn style

### LINKED ISSUE #7: The navbar class for header
- [x] the "`navbar`" class

### PR DESCRIPTION
**Pull request by inggritenovaleta**
I added the style for the navbar and the button Apply Now. I have a difficulty in putting the logo on the left side, please guide me. Thank you in advance

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -57,8 +137,17 @@
     color: rgb(255, 255, 255);
     text-align: center;
     max-width: none;
- 
-}
 
+}
 
+.container {
+    margin: 0px;
```
Comment (rago89) [inline public/css/style.css:277]: try to put the margin and the overflow auto

**Review by rago89 — COMMENTED** [review]
i have some suggestion to see if it works with the image posotioning

File: public/css/style.css (hunk 1 above)
Comment (inggritenovaleta) [reply public/css/style.css:277]: okay I will try that :) thank you for your suggestion. I will let you know:)

File: public/css/style.css [hunk 2]
```diff
@@ -10,6 +27,69 @@
...
         flex-direction: row;
 }
+.logo{
+    display:block;
+    align-items:center;
+    height: 60%;
+}
+
+/* navbar */
+.navbar {
```
Comment (rago89) [inline public/css/style.css]: on this one try to put .flex class

**Review by rago89 — COMMENTED** [review]
add a suggestion to fix the problem positioning the  logo

File: public/css/style.css (hunk 2 above)
Comment (inggritenovaleta) [reply public/css/style.css]: @rago89 I made some adjustment - please kindly review. It shows like this. This is the best I can do, do you have any suggestion? thank you in advcance
image

**Review by rago89 — COMMENTED** [review]
Hi, it's better now can you please take a look at the merging conflicts

**Pull request merged**